app/src/androidTest/java/com/yacgroup/yacguide/database/RegionDaoTests.kt

```diff
@@ -107,26 +89,56 @@ class RegionDaoTests {
     @Test
     fun getRegion_regionAvailable_returnsRegion() = runBlockingTest {
         val region = REGIONS.first()
-
         assertTrue(_regionDao.getRegion(region.id) == region)
     }
+}
+
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
```

Comment:
Note: I've grouped the tests to a "getter" and a "deletion" class, so that don't need to clear and refill the test database (which is obviously necessary for deletion test cases) for each test case when just testing database getter requests.